utils/server-instance.js

```diff
@@ -78,6 +78,48 @@ class ServerInstance {
       res.send(JSON.stringify(req.cookies));
     });
 
+    // Used by the following two routes.
+    function respondWithScriptMatchingPattern(pattern, res) {
+      const repoRoot = path.join(__dirname, '..');
+      const devScript = glob.sync(pattern, {
+        cwd: repoRoot,
+        root: repoRoot,
+      });
+
+      if (devScript.length === 0) {
+        const errMsg = `No built module found for pattern '${pattern}'.`;
+        console.log(errMsg);
+        res.status(500).send(errMsg);
+        return;
+      }
+
+      // We can't send a redirect here, as they're not allowed for resources
+      // pulled in via importScripts().
+      res.setHeader('Content-Type', 'application/javascript');
+      res.send(fs.readFileSync(devScript[0]));
+    }
+
+    // Used to return the latest iife bundle for a given package, without
+    // having to specify the version string.
+    // This is used within unit tests.
+    this._app.get('/__test/bundle/:pkg', function(req, res) {
+      const pkg = req.params.pkg;
+      const pattern = `packages/${pkg}/build/importScripts/*dev*.js`;
+      respondWithScriptMatchingPattern(pattern, res);
+    });
+
+    // Used to return the latest iife bundle for a given package, without
+    // having to specify the version string, for packages that are installed
+    // via lerna.
+    // This is used within unit tests.
+    this._app.get('/__test/lernaBundle/:parentPkg/:pkg', function(req, res) {
```

Comment:
I *thought* that this was needed because, e.g., the `packages/sw-runtime-caching/build` wouldn't exist when a single-project build, like `gulp build --project='sw-cache-expiration'` was performed.

Now after experimenting, I realize that `lerna bootstrap` will build that, (and also symlink), so I can drop it. Thanks for calling that out.